indexer/src/main.ts

```diff
@@ -0,0 +1,35 @@
+import { TypeormDatabase } from '@subsquid/typeorm-store';
+
+import { processor } from './processor';
+import { config } from './config';
+import { EventsProcessing } from './processing/events.processing';
+import {Collection} from "./model";
+
+processor.run(new TypeormDatabase(), async (ctx) => {
+  const processing = new EventsProcessing(ctx.store);
+  for (const block of ctx.blocks) {
+    const events = [
+      ...block.events,
+      ...block.extrinsics.flatMap((e) => e.events),
+    ].filter((e) => e.args && e.args.message);
```

Comment:
Not necessary if `extrinsic` is removed. Perhaps you can filter events by the `source` field to have events only from programs that will be indexed